modules/learn/pages/clusters-and-availability/size-limitations.adoc

```diff
@@ -71,4 +72,9 @@ If the execution time for a document exceeds the specified value, the execution
 | Administrator-defined
 | The maximum permitted numbers of connections permitted for memcached connections, and for system-user connections, for the cluster.
 These numbers are established by means of the REST API: see xref:rest-api:rest-manage-cluster-connections.adoc[Managing Cluster Connections], for information.
+
+| Unsigned 64 bit integer
+| latexmath:[2^{63} - 1]
+| The integer is truncated if larger, due to floating-point arithmetic approximation.
```

Comment:
I'd phrase it as something like:
Integers outside this range are stored as 64-bit floating-point values and are subject to approximation inherent in the IEEE floating-point format.